docs/reference/support.md

```diff
@@ -42,27 +43,34 @@ The Ionic team has compiled a set of recommendations for using the Ionic Framewo
 
 | Framework | Minimum Angular Version | Maximum Angular Version | TypeScript |
 | :-------: | :---------------------: | :---------------------: | :--------: |
-|    v6     |           v12           |          v14.x^         |    4.0+    |
-|    v5     |          v8.2           |          v12.x          |    3.5+    |
-|    v4     |          v8.2           |          v11.x          |    3.5+    |
-|    v3     |         v5.2.11         |         v5.2.11         |   2.6.2    |
+| v7        | v13                     | v14.x                   | 4.4+       |   
+| v6        | v12                     | v14.x^                  | 4.0+       |
+| v5        | v8.2                    | v12.x                   | 3.5+       |
+| v4        | v8.2                    | v11.x                   | 3.5+       |
+| v3        | v5.2.11                 | v5.2.11                 | 2.6.2      |
 
 > ^ Angular 14.x support was added in Ionic v6.1.9.
 
+**Angular 13+ Support On Older Versions of iOS**
+
+Angular's support policy for iOS is the two most recent major versions. This means that changes to your Angular project may be necessary to use Ionic Angular v4-v6 on iOS 13. To support iOS 13, change the project `target` specified in `compilerOptions` in the tsconfig.json to `es5`. Without this change an error of `Unexpected token '.' in promiseReactionJob` will occur on app startup in iOS 13.
```

Comment:
Should the Ionic versions also include v7?

Reply:
Ionic v7 does not support iOS 13, so this note does not apply. I'll make this clearer in the docs.

Reply:
Oh actually we call out Ionic v4-v6. Can you think of another another way I could make this clearer?

Reply:
Maybe something like this:
> Note that later versions of Ionic do not support iOS 13; see [mobile support table here](/docs/reference/browser-support).

Now that this section is on a different page from said table, the context is a bit lost.

Reply:
Would it be better if I just keep the Angular note where it was before? It just seemed odd to an Angular 13 mention on a page that only talks about browsers/platforms.

Reply:
Nah, I agree that this is the best place for it since it only pertains to Angular devs. I think the only way to get this completely clean would be to merge the two support pages into one, but that seems a little drastic 😆